lucille-core/src/main/java/com/kmwllc/lucille/indexer/SolrIndexer.java

```diff
@@ -85,8 +64,9 @@ public boolean validateConnection() {
       log.error("Null response when pinging solr");
       return false;
     }
-    if (response.getStatus() != 0) {
-      log.error("Non zero response when pinging solr: " + response.getStatus());
+    Integer status = (Integer) ((SimpleOrderedMap) response.get("responseHeader")).get("status");
+    if (status != 0) {
+      log.error("Non zero response when pinging solr: " + status);
```

Comment:
So we're not pinging anymore... might want to change the log message to say "checking cluster status"